Extensions/BitmapText/JsExtension.js

```diff
@@ -0,0 +1,656 @@
+      .setFunctionName('getWordWrap');
+
+    object
+      .addCondition(
+        'GetWrappingWidth',
+        _('Wrapping width'),
+        _(
+          'Compare the width, in pixels, after which the text is wrapped on next line.'
+        ),
+        _('the wrapping width'),
+        _('Style'),
+        'res/actions/wordWrap24.png',
+        'res/actions/wordWrap.png'
+      )
+      .addParameter('object', _('Bitmap text'), 'BitmapTextObject', false)
+      .useStandardRelationalOperatorParameters('number')
+      .getCodeExtraInformation()
+      .setFunctionName('getWrappingWidth');
+
+    object
+      .addAction(
+        'SetText',
+        _('Text'),
+        _('Modify the text of a Bitmap text object.'),
+        _('the Bitmap text'),
+        '',
+        'res/actions/text24.png',
+        'res/actions/text.png'
+      )
+      .addParameter('object', _('Bitmap text'), 'BitmapTextObject', false)
+      .useStandardOperatorParameters('string')
+      .getCodeExtraInformation()
+      .setFunctionName('setText')
+      .setGetter('getText');
+
+    object
+      .addAction(
+        'SetColor',
+        _('Color'),
+        _('Change the color of the text.'),
+        _('Change color of _PARAM0_ to _PARAM1_'),
+        _('Style'),
+        'res/actions/color24.png',
+        'res/actions/color.png'
+      )
+      .addParameter('object', _('Bitmap text'), 'BitmapTextObject', false)
+      .addParameter('color', _('Color'), '', false)
+      .setDefaultValue('0;0;0')
+      .getCodeExtraInformation()
+      .setFunctionName('setColor');
+
+    object
+      .addAction(
+        'SetOpacity',
+        _('Opacity'),
+        _('Modify the opacity of a Bitmap text object.'),
+        _('the opacity'),
+        '',
+        'res/actions/opacity24.png',
+        'res/actions/opacity.png'
+      )
+      .addParameter('object', _('Bitmap text'), 'BitmapTextObject', false)
+      .useStandardOperatorParameters('number')
+      .getCodeExtraInformation()
+      .setFunctionName('setOpacity')
+      .setGetter('getOpacity');
+
+    object
+      .addAction(
+        'SetFontSize',
+        _('Font size'),
+        _('Modify the font size of a Bitmap text object.'),
+        _('the font size'),
+        '',
+        'res/actions/font24.png',
+        'res/actions/font.png'
+      )
+      .addParameter('object', _('Bitmap text'), 'BitmapTextObject', false)
+      .useStandardOperatorParameters('number')
+      .getCodeExtraInformation()
+      .setFunctionName('setFontSize')
+      .setGetter('getFontSize');
+
+    object
+      .addAction(
+        'SetFontResourceName',
+        _('Font'),
+        _('Change the font used by the Bitmap text object.'),
+        _('Set the font of _PARAM0_ to _PARAM1_'),
+        _('Style'),
+        'res/actions/font24.png',
+        'res/actions/font.png'
+      )
+      .addParameter('object', _('Bitmap text'), 'BitmapTextObject', false)
+      .addParameter('font', _('Font resource'), '', false)
```

Comment:
We'll probably need here a `bitmapFont` parameter. Add a TODO, we can add support for this parameter type in the IDE later :)

Reply:
Actually I see you made a field for it already :) This should probably be `bitmapFontResource` then? :)